suites/test_assembly_content.py

```diff
@@ -0,0 +1,69 @@
+import sys
+from helpers import base
+import lemoncheesecake.api as lcc
+from lemoncheesecake.matching import *
+
+from helpers import constants
+import requests
+from fixtures import fixture
+from polling2 import poll
+from helpers import utilities
+import urllib.parse
+
+sys.path.append("..")
+
+assembly_title_prefix = base.config_reader('test_repo', 'assembly_prefix')
+assembly_prefix = base.config_reader('test_repo', 'assembly_content_prefix')
+repo_name = base.config_reader('test_repo', 'repo_name')
+
+
+@lcc.suite(description="Suite: Verify contents of published assembly", rank=4)
+class test_assembly_content:
+  api_auth = lcc.inject_fixture("api_auth")
+
+  path_for_assembly = ""
+  request_url = ""
+
+  @lcc.test("Verify response of assembly variant api")
+  def verify_assembly_content(self, api_auth, setup_test_products):
+      self.variant = utilities.read_variant_name_from_pantheon2config()
+      lcc.log_info(str(self.variant))
+      self.variant = str(self.variant)
+      self.path_for_assembly = utilities.select_first_item_from_search_results(fixture.url, assembly_prefix)
+      res, product_name_uri = utilities.add_metadata(fixture.url, self.path_for_assembly, self.variant, api_auth, setup_test_products, content_type="assembly")
+      print(res.content)
+      utilities.publish_content(fixture.url,self.path_for_assembly, self.variant, api_auth)
+
+      assembly_uuid = utilities.fetch_uuid(fixture.url, self.path_for_assembly, self.variant)
+      published_assembly_url = fixture.url + "api/assembly/variant.json/" + assembly_uuid
+      print("published assembly url: \n" + published_assembly_url)
+      lcc.log_info("Published Assembly api endpoint: %s" % published_assembly_url)
+      data_from_published_assembly = api_auth.get(published_assembly_url)
+      check_that("The /api/assembly/variant.json/<assembly_uuid> endpoint for a published assembly",
+                 data_from_published_assembly.status_code, equal_to(200))
+
+      print("Response from published assembly API endpoint: \n" + str(data_from_published_assembly.content))
+      check_that("The response is ", data_from_published_assembly.json()["message"], equal_to("Assembly Found"))
+      check_that("The title of the assembly ", data_from_published_assembly.json()["assembly"]["title"],
+                 contains_string(assembly_prefix))
+      check_that("The status of the assembly ", data_from_published_assembly.json()["assembly"]["status"],
+                 equal_to("published"))
+      check_that("The variant uuid of the assembly", data_from_published_assembly.json()["assembly"]["variant_uuid"],
+                 equal_to(assembly_uuid))
+      check_that("The uuid of the assembly", data_from_published_assembly.json()["assembly"]["uuid"], equal_to(assembly_uuid))
+      check_that("The abstract of the assembly", data_from_published_assembly.json()["assembly"]["description"],
```

Comment:
This check should be different if we are taking into consideration an assembly which doesn't have the "abstract" specified in .adoc right? but given ther document has it, the check seems to be valid I believe.